### PR TITLE
Update private-images.md

### PR DESCRIPTION
**Pull request by jdewinne**

### CODE REVIEW

**Comment by AmberAlston** [comment]
@jdewinne is this still in-progress or ready for review? If ready for review, can you add a reviewer please?

**Review by tdensmore — APPROVED** [review]
LGTM. Thanks for adding the username/password and the token/token options.

File: content/vendor/packaging/private-images.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ title: Private Images and Registry Credentials
 weight: 20200
 ---
 
+When building your application, you have the option to use the Replicated private registry, or any supported external private or public registry.
```
Comment (marccampbell) [inline content/vendor/packaging/private-images.md:8]: I think the second comma shouldn't be here.

**Review by sgalsaleh — APPROVED** [review]

File: content/vendor/packaging/private-images.md [hunk 2]
```diff
@@ -67,6 +71,27 @@ However, the same secret will be added to those PodSpecs as well.
 
 > KOTS [Application](/reference/v1beta1/application/) deployments are supported via image tags in all use cases. KOTS has limited support for deploying via image digests. Use of image digests are only supported for fully online installs where all images can be pulled from the Replicated registry, a public repo, or proxied from a private repo via the Replicated registry.
 
+## Replicated Private Registry
+
+When using the Replicated Private Registry, you have 2 options to connect with the `registry.replicated.com` container registry
```
Comment (tdensmore) [inline content/vendor/packaging/private-images.md:76]: how about a colon after "registry"?

"When using the Replicated Private Registry, you have 2 options to connect with the `registry.replicated.com` container registry:"

**Review by tdensmore — CHANGES_REQUESTED** [review]
LGTM with the single suggestion

**Pull request merged**